### PR TITLE
Fixes # linux build

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -48,6 +48,9 @@ before_test:
   - ps: $env:Path += ";C:\Program Files (x86)\Pandoc\"
   - pandoc -v
 
+after_build:
+  - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%]"
```
Comment (msevestre) [inline appveyor.yml]: @Yuri05 My hope is that this will be triggered and replace the tar ball already created...to see

File: rakefile.rb [hunk 2]
```diff
@@ -16,8 +16,59 @@
   copy_files_to_lib_folder 
 end
 
+# This task is temporary until we have an automated linux buold
+task :create_linux_build, [:product_version, :build_dir] do |t, args|
```
Comment (msevestre) [inline rakefile.rb]: @Yuri05 Pretty straight forward using some of the method we have already

File: appveyor.yml (hunk 1 above)
Comment (msevestre) [reply appveyor.yml]: does not work quite yet. This is to early

**Comment by msevestre** [comment]
@Yuri05 I have no idea why it does not work on appveyor. Scripts works locally no problem. on appveyor, I get some kind of broken pipe error. This happens when zipping the folder

**Comment by msevestre** [comment]
@Yuri05 https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r#L1019

I have tried everything. Forward \ or / with / at the end or not. I have no idea why 

tar xzf C:/projects/ospsuite-r/ospsuite_9.0.47.tar.gz -C C:/projects/ospsuite-r/temp
tar cvzf C:/projects/ospsuite-r/ospsuite_9.0.47.tar.gz C:/projects/ospsuite-r/temp/
tar: Removing leading `C:/' from member names
C:/projects/ospsuite-r/temp/
C:/projects/ospsuite-r/temp/ospsuite/
tar (child): gzip: Cannot exec: No such file or directory
tar (child): Error is not recoverable: exiting now
C:/projects/ospsuite-r/temp/ospsuite/build/
C:/projects/ospsuite-r/temp/ospsuite/build/vignette.rds
C:/projects/ospsuite-r/temp/ospsuite/DESCRIPTION
C:/projects/ospsuite-r/temp/ospsuite/inst/
C:/projects/ospsuite-r/temp/ospsuite/inst/doc/
C:/projects/ospsuite-r/temp/ospsuite/inst/doc/introduction-ospsuite.html
tar: C\:/projects/ospsuite-r/ospsuite_9.0.47.tar.gz: Cannot write: Broken pipe
tar: Child returned status 2
tar: Error is not recoverable: exiting now
rake aborted!
Command failed with status (2): [tar cvzf C:/projects/ospsuite-r/ospsuite_9...]

**Comment by msevestre** [comment]
This is driving me crazy
I even created another repo where I am just doing the tar stuff and it works
image


I just don't get it

**Comment by msevestre** [comment]
@Yuri05 I am not creating a zip file for linux as tar does not work. Please let me know if this works underlinux

Logic: 
* Take tar file
* untar
* remove all natives dll
* replace with lib*.so
* replace netstandard dlls for funcparser and simmodel
* zip folder under xxx_linux.zip

image


Can you try and let me know if this works?
https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/build/artifacts

if not, you might want to think about the freaking tar issue becuase I am out of idea:)

**Comment by Yuri05** [comment]
ok, after conversion from .zip to .tar.gz: works fine.
I think we can merge for now.

**Comment by msevestre** [comment]
going to add the following
rename build to ubuntu instead of linux
And try one last thing before merging

**Comment by Yuri05** [comment]
lol yuhuu :)

**Pull request merged**